webapp/platform/types/src/config.ts

```diff
@@ -64,6 +64,7 @@ export type ClientConfig = {
     EnableCustomTermsOfService: string;
     EnableDeveloper: string;
     EnableDiagnostics: string;
+    EnableDesktopLandingPage: 'true' | 'false';
```

Comment:
We've started specifying newer ones with their actual values since they're defined as true or false on the server. I don't have a good reason for why we haven't gone back and done that with the other ones, but we could certainly do that at any point